lib/worker-pipeline.js

```diff
@@ -112,25 +112,101 @@ class AssetComputeWorkerPipeline {
         // TODO we will need to add support for multiple renditions
         // TODO we have to integrate options into our plan and support everything we support now
         
-        debug("Preparing plan for rendition creation...");
+        debug("compute: Preparing plan for rendition creation...");
         engine.refinePlan(plan, input, params.renditions[0]);
-        debug("Refined plan to create rendition");
+        debug("compute: Refined plan to create rendition: ", plan.toString());
 
-        debug("Running pipeline...");
+        debug("compute: Running pipeline...");
         const pipelineOutput = await engine.run(plan);
-        debug("Pipeline finished running");
-        debug("######## pipelineOutput:", pipelineOutput);
+        debug("compute: Pipeline finished running");
+        debug("compute: pipeline output:", pipelineOutput);
```

Comment:
note: I noticed some of our workers run in earlier nodejs versions where `debug` does not support comma-separated logs. we should probably stick to template literals for this reason